main/src/vtr_radar/src/pipeline.cpp

```diff
@@ -171,6 +211,17 @@ void RadarPipeline::onVertexCreation_(const QueryCache::Ptr &qdata0,
     CLOG(DEBUG, "radar.pipeline") << "Saved raw pointcloud to vertex" << vertex;
   }
 
+  // to add another function to save the radar b_scan_img to the vertex
+  // radar images
+  if(config_->save_radar_images)
```

Comment:
You should add a check to the condition whether `qdata->scan_msg` is valid. In offline runs it will not be.

Reply:
if(qdata->scan_msg) will do?

Reply:
I think it needs to `qdata->scan_msg.valid()`

Reply:
This will do actually to check if it was set as a pointer
I tested it